### PR TITLE
update coherence forward/backward

### PR DESCRIPTION
**Pull request by ludwigVonKoopa**

### CODE REVIEW

File: src/py_eddy_tracker/observations/groups.py [hunk 1]
```diff
@@ -87,11 +88,55 @@ def advect(x, y, c, t0, n_days):
     return t, x, y
 
 
-def particle_candidate(x, y, c, eddies, t_start, i_target, pct, **kwargs):
+def create_particles(eddies, step):
+    """create particles only inside speed contour. Avoir creating too large numpy arrays, only to me masked
```
Comment (AntSimi) [inline src/py_eddy_tracker/observations/groups.py:92]: Avoir?

**Review by AntSimi — CHANGES_REQUESTED** [review]
some warning with flake8
./src/py_eddy_tracker/observations/network.py:1476:128: E501 line too long (139 > 127 characters)
./src/py_eddy_tracker/observations/network.py:1539:128: E501 line too long (139 > 127 characters)
./examples/07_cube_manipulation/pet_fsle_med.py:30:56: W291 trailing whitespace
./examples/16_network/pet_follow_particle.py:19:1: F401 'py_eddy_tracker.poly.group_obs' imported but unused

File: src/py_eddy_tracker/observations/groups.py (hunk 1 above)
Comment (ludwigVonKoopa) [reply src/py_eddy_tracker/observations/groups.py]: mistake

**Comment by ludwigVonKoopa** [comment]
better create_particles

some differences exists with new version, as old version sometimes created double particles (2 different particles with same lon/lat).
Bounding box were squared and chosen a little bit over min/max, so if 2 eddy were close enough and with proper shape, there was chances to create double particles

**Review by AntSimi — APPROVED** [review]

**Pull request merged**